igcviewer.js

```diff
@@ -206,6 +206,7 @@ var ns = (function($) {
     var finishTime;
     var finishAlt;
     var bestSoFar = 0;
+    var startIndexBest;
```

Comment:
On 08/04/16 22:26, Alistair Green wrote:
>
> In igcviewer.js 
> <https://github.com/GlidingWeb/IGCWebView/pull/7#discussion_r59091673>:
>
> > @@ -206,6 +206,7 @@ var ns = (function($) {
> >      var finishTime;
> >      var finishAlt;
> >      var bestSoFar = 0;
> > +    var startIndexBest;
>
> This variable is never used. Did you get it mixed up with 
> startIndexLatest?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/GlidingWeb/IGCWebView/pull/7/files/b13b4a1e52d690b129e31643e9b9b81e076316f7#r59091673>
>
Er....

Yes and no.  StartIndexBest was originally intended as a placeholder for 
a previous start time when there is a subsequent start, until the 
distance flown after the later start exceeds that flown after the 
earlier start.  After a bit of thought it became clear that it wasn't 
needed as there was no need to reset the original start time at that 
point.  I then forgot to delete it- I will now. startIndexLatest is the 
last crossing of the start line, regardless of whether an intended start 
or not.

Regards